package.json

```
@@ -4,6 +4,37 @@
   "description": "Reactive Extensions for modern JavaScript",
   "main": "index.js",
   "sideEffects": false,
+  "exports": {
+    ".": {
+      "node": "./index.js",
+      "default": "./_esm5/index.js"
```

can you please also include `es2015` version here and in all the other entry-points?
```suggestion
      "default": "./_esm5/index.js",
      "es2015": "./_esm2015/index.js"
```

---

we should also forward-port this change to rxjs@7, @crisbeto could you please send a separate PR for that? thank you

---

This change would fix https://github.com/ReactiveX/rxjs/issues/6321 as well.

---

Done. Will send a PR for the v7 branch ASAP.

---

https://github.com/ReactiveX/rxjs/pull/6614